File: WebUI/admin-app/src/app/models/sportObject.ts
export interface ISportObjectsVm {
  sportObjects: ISportObject[];
  sportObjectsCount: number;
}

export interface ISportObject {
  id?: number;
  email: string;
  name: string;
  address: string;
  phone: string;
  description: string;
  isPayed: boolean;
  isPremium: boolean;
  sportId: number;
  sport: ISport;
  cityId: number;
  city: ICity;
  prices: IPrice[];
  workingHours: IWorkingHour[];
  reviews: IReview[];
  reservations: IReservation[];
}

interface ISport {
  id: number;
  name: string;
}

interface ICity {
  id: number;
  name: string;
}

interface IPrice {
  id: number;
  pricePerHour: number;
  timeFrom: string;
  timeTo: string;
}

interface IWorkingHour {
  id: number;
  day: number;
  openTime: string;
  closeTime: string;
}

interface IReview {
  username: string;
  rating: number;
  comment: string;
  createdAt: Date;
}

interface IReservation {
  id: number;
  startTime: string;
  endtTime: string;
  date: Date;
  user: IUser;
  status: IReservationStatus;
}

interface IUser {
  id: string;
  username: string;
}

interface IReservationStatus {
  id: number;
  status: string;
}

File: Domain/Entities/SportObject.cs
using System.Collections.Generic;

namespace Domain.Entities
{
  public class SportObject
  {
    public int Id { get; set; }
    public string Email { get; set; }
    public string Name { get; set; }
    public string Address { get; set; }
    public string Phone { get; set; }
    public string Description { get; set; }
    public bool IsPayed { get; set; }
    public bool IsPremium { get; set; }
    public int SportId { get; set; }
    public int CityId { get; set; }

    public virtual ICollection<Review> Reviews { get; set; }
    public virtual ICollection<WorkingHour> WorkingHours { get; set; }
    public virtual ICollection<Price> Prices { get; set; }

    public virtual Sport Sport { get; set; }
    public virtual City City { get; set; }
  }
}